README.md

```diff
@@ -102,6 +102,44 @@ unlock()
 ```
 
 
+### Install
+
+`installDevice<Variant>` tasks are available by default just to install the app. Plugin also supports `install` dsl to define custom installation tasks.
```

Comment:
I'd suggest:
"The plugin also supports an `install` dsl to define custom installation tasks."